cha/implementaion/initial/ci.tex

```diff
@@ -1 +1,52 @@
-\section{Continous Integration}
+\section{Automated System Building with Continuous Integration}
+%We want to be assisted in integrating newly developed features etc.
+%Also, automate tests (dev env / target) and docs
+%Android APK availability
+%Hand in hand with agile
+While implementing our Android application, it is important that we do not get disconnected from the end product, albeit an unfinished one.
```

Comment:
Do you mean that it is important that we do not lose focus? not sure what you mean by disconnected